scripts/benchmark_cuda_end_to_end.py

```
@@ -0,0 +1,865 @@
+    print(report, end="")
+    print(f"\nMachine-readable results: {result_path}")
+    if not args.keep_outputs:
+        shutil.rmtree(output / "outputs")
+    return 0
+
+
+def report_results(args: argparse.Namespace) -> int:
+    results = [
+        json.loads(path.read_text(encoding="utf-8")) for path in args.results
+    ]
+    for path, result in zip(args.results, results):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)